src/content/docs/apm/agents/nodejs-agent/api-guides/guide-using-nodejs-agent-api.mdx

```diff
@@ -27,7 +27,7 @@ Other resources:
 To use the Node.js agent API, make sure you have the [latest Node.js agent release](/docs/release-notes/agent-release-notes/nodejs-release-notes). In addition, see:
 
 * [Node.js agent API requirements](/docs/agents/nodejs-agent/supported-features/nodejs-agent-api)
-* [Getting started procedures on GitHub](http://newrelic.github.io/node-newrelic/docs/index.html#getting-started)
+* [Getting started procedures on GitHub](http://newrelic.github.io/node-newrelic/index.html#getting-started)
```

Comment:
@bizob2828 For some reason this anchor tag link doesn't work. As far as I can tell from the HTML code on this GitHub.io page, the headers don't actually have IDs, which means that can't be linked to directly.

Reply:
we must've changed structure around. I can remove the anchor

Reply:
I'm happy to do it.